pkg/cmd/pr/close/close.go

```diff
@@ -35,9 +35,9 @@ func NewCmdClose(f *cmdutil.Factory, runF func(*CloseOptions) error) *cobra.Comm
 	}
 
 	cmd := &cobra.Command{
-		Use:   "close {<number> | <url> | <branch>}",
+		Use:   "close [<number> | <url> | <branch>]",
 		Short: "Close a pull request",
-		Args:  cobra.ExactArgs(1),
```

Comment:
This might have not been apparent, but we wanted to force the user to explicitly supply the argument to `close` instead of taking the current PR implicitly. This made it harder to close the current PR by accident, even though it was inconsistent with other pr commands where the argument was optional.